platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxCountActivity.kt

```diff
@@ -89,7 +89,8 @@ class QueryRenderedFeaturesBoxCountActivity : AppCompatActivity() {
     override fun onStart() {
         super.onStart()
         mapView.onStart()
-        if (mapboxMap != null) {
+
```

Comment:
@louwers I suggest checking both: mapboxMap != null && mapboxMap.isInitialized

Reply:
@ovivoda Kotlin is null safe right?. It is not nullable so it cannot be null.

Reply:
@louwers yes :)

Reply:
https://bignerdranch.com/blog/kotlin-when-to-use-lazy-or-lateinit/